hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/ratis/TestRatisHelper.java

```diff
@@ -1,122 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.hadoop.hdds.ratis;
-
-import org.apache.hadoop.hdds.conf.OzoneConfiguration;
-import org.apache.ratis.conf.RaftProperties;
-import org.junit.Assert;
-import org.junit.Test;
-
-/**
- * Test RatisHelper class.
- */
-public class TestRatisHelper {
```

Comment:
Is this removed intentionally?  The methods being tested are still present in RatisHelper.

Reply:
No, it was not intentional. I restored.